docs/source/guides/versioning.md

```diff
@@ -69,14 +75,16 @@ type Query {
 }
 ```
 
-GraphQL-aware tools, like GraphQL Playground and GraphiQL, use this information to assist developers in making the right choices.  These tools will:
+GraphQL-aware client tooling, like GraphQL Playground and GraphiQL, use this information to assist developers in making the right choices.  These tools will:
 
 * Provide developers the helpful deprecation message which instructs them to use the new name.
```

Comment:
"Provide developers the helpful deprecation message.." => "Provide developers with the helpful deprecation message..."